tests/unit/test_service_steps.py

```diff
@@ -661,3 +662,34 @@ def test_emr_modify_instance_group_by_name_step_creation():
         'End': True
     }
 
+
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_databrew_start_job_run_step_creation_sync():
+    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run', parameters={
```

Comment:
```suggestion
    step = GlueDataBrewStartJobRunStep('Start Glue DataBrew Job Run - Sync', parameters={
```

minor: just something to indicate what we're testing here